src/include/ftp-server/dialogs.rb

```diff
@@ -121,23 +121,23 @@ def StartMode
       )
       Ops.set(
         result,
-        "get_service_start_via_xinetd",
+        "get_service_start_via_socket",
         fun_ref(method(:"started_via_socket?"), "boolean ()")
       )
       Ops.set(
         result,
-        "set_service_start_via_xinetd",
+        "set_service_start_via_socket",
         fun_ref(method(:"start_via_socket="), "void (boolean)")
       )
       # TRANSLATORS: Radio selection
       Ops.set(result, "start_auto_button", _("&When booting"))
       Ops.set(result, "start_manual_button", _("&Manually"))
-      Ops.set(result, "start_xinetd_button", _("Via &xinetd"))
+      Ops.set(result, "start_socket_button", _("Via &socket"))
```

Comment:
isn't that defaults? so maybe we don't need to set it?

Reply:
Yep, probably is not needed, I basically touched because when was testing I was surprised because xinetd button was still there.

Let me check if is something we actually can remove.

Thnx ;)

Reply:
@jreidinger I see just differences in the `start_auto_button` and in `start_manual_button`. In the last one just the shortcut is not proposed and being "During boot" the one for start_auto_button.

Reply:
![defaults](https://user-images.githubusercontent.com/7056681/36021447-b530ee52-0d7d-11e8-82a8-ab10c42f5212.png)

Reply:
![overrideddefaults](https://user-images.githubusercontent.com/7056681/36021492-d963411c-0d7d-11e8-8cf6-8f6974c2259c.png)

Reply:
well, I vote to unify it when possible

Reply:
Probably it was done because the call to CWMServiceStart.AutoStartHelpSocketTemplate, as them are independent and you have to specify concretely the labels, so if you modify the defaults them it will be inconsistent.